src/Thenewboston.Tests/Bank/Api/InvalidBlockServiceTests.cs

```diff
@@ -1,7 +1,93 @@
-﻿namespace Thenewboston.Tests.Bank.Api
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using Newtonsoft.Json;
+using Thenewboston.Bank.Api;
+using Thenewboston.Bank.Models;
+using Thenewboston.Common.Http;
+using Xunit;
+
+namespace Thenewboston.Tests.Bank.Api
 {
     public class InvalidBlockServiceTests
     {
-        static 
+        public class GetInvalidBankBlocksAsync
```

Comment:
Tests will unfortunately need to be updated to work with PaginatedResponseModel being returned.

Reply:
Tests Updated